easyDiffractionApp/Gui/Components/AnalysisConstraints.qml

```diff
@@ -91,15 +104,15 @@ EaComponents.TableView  {
             width: EaStyle.Sizes.fontPixelSize * 3
             headerText: "Use"
             checked: model.enabled
-            onToggled: ExGlobals.Constants.proxy.toggleConstraintByIndex(model.index, checked)
+            onToggled: ExGlobals.Constants.proxy.fitLogic.toggleConstraintByIndex(model.index, checked)
```

Comment:
toggling does not necessarily need `checked` - you will just revert whatever is in the current model.
Logics should take care of this without knowing what `checked` is.